README.md

```diff
@@ -20,27 +20,26 @@ For more information and to learn more about how to work with this pallet, the f
 A basic sample of the Move module and the Move script is shown below.
 
 ```move
-module DeveloperBob::CarPool {
+module DeveloperBob::CarWash {
     // ...
 
-    /// Requests the rent of a car on the given day and between given daytime hours.
-    /// Returns `true` in case of success, otherwise `false`.
-    public fun rent_car(account: &signer, day: u16, lease_time: u16, token_limit: u128): bool {
+    /// Buys a washing coin for the car wash. Therfor, `COIN_PRICE` will be withdrawn from the user's account.
```

Comment:
```suggestion
    /// Buys a washing coin for the car wash. Therefore, `COIN_PRICE` will be withdrawn from the user's account.
```

Reply:
Done, but will add this in another commit, because I wanted one single commit for the tutorial part and the other for the included example code and its unit-test